src/components/shared/scheduler/SchedulerEditor.vue

```diff
@@ -238,52 +122,13 @@
             CroudDatepicker,
         },
 
+        filters: {
```

Comment:
Could this not be done with css?

Reply:
Sure, but I was refactoring whilst trying to match the existing snapshots. I can move it to a css rule if we want, but that won't be testable.

Reply:
ah, gotcha.

Reply:
Cool, are you happy for this to be merged in?